scripts/setup-helper-functions.sh

```diff
@@ -108,15 +108,15 @@ function get_cxx_flags {
   case $CPU_ARCH in
 
     "arm64")
-      echo -n "-mcpu=apple-m1+crc -std=c++17 -fvisibility=hidden"
+      echo -n "-mcpu=apple-m1+crc -fvisibility=hidden"
```

Comment:
@assignUser Our setup script functions use `get_cxx_flags` to build the dependencies. Should we retain this?

Reply:
No, as it breaks big int support (big int is an extension and c++17 is without extensions, this is not enforced on all compilers but I ran into it with gcc ). 

Dependencies will handle their own minimum standard and I don't see a need to enforce that and CMake also has ways for us to do it without explicitly adding the flag for velox. Also with any remotely recent compiler 17 will be the implicit default anyway (gcc>=11).

Reply:
In fact, this was a problem for certain dependencies when switching to Clang. The GCC C compiler ignored `-std=c++17` but Clang threw an error. 
You can update `setup-centos9.sh` line 35: https://github.com/facebookincubator/velox/blob/52c1daaa10a896d2e96f322b710ed5860dbd4e53/scripts/setup-centos9.sh#L35 and set CFLAGS to CXXFLAGS as well.

Reply:
@czentgr intersting! I think we can just remove it for this and my above reasons. My temp CI container is built with this script and doesn't show any erros when building the dependencies.

I actually think we can remove most of the flag things for dependencies except for folly where we need to match simd flags (or atleast have sse for their F14map as an avx implementation is missing) but we can do that in a follow up as I haven't really tested that yet.

Reply:
> Dependencies will handle their own minimum standard

This makes sense.